src_data/html/doiHeader.mustache

```diff
@@ -147,7 +147,8 @@
 </div>
   </div>
       <div class="l-region l-region--navigation">
-	  <div id="languageChanger"><p>English/Espa&#241;ol</p></div>
+	  <a href="https://www.doi.gov/water/owdi.cr.drought/en/index.html"><div id="english" class="languageChanger"><p>English</p></div></a>
```

Comment:
Let me look at the render_html script, because we can probably add a variable to the context that allows us to do an if in the template here.